.github/workflows/unittest.yml

```
@@ -1 +1,177 @@
+        ls -la DotenkoV3/Config/Config.swift DotenkoV3/Info.plist DotenkoV3/GoogleService-Info.plist 2>/dev/null || echo "Some files missing"
+        echo "ℹ️ AdMob設定はConfig.swiftで管理されています"
+    
+    # Xcodeバージョンの設定
+    - name: Setup Xcode
+      uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: '15.4'
+    
+    # CocoaPodsのキャッシュ設定
+    - name: Cache CocoaPods
+      uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)